### ISSUE TITLE
python support

### ISSUE THREAD

**Issue by BMaxV** [clockworklabs/SpacetimeDB#949]
Hi,

I saw this docs page

https://spacetimedb.com/docs/sdks/python/quickstart

and it's not really clear how the server is supposed to work, do you have a minimal, working "hello world", server client thing example?

**Comment by hemna** [comment]
yes please add python support.  I'm not interested in rust or typescript at all.

**Comment by Lynden-Sylvester** [comment]
I'd love to see python support. It'd be extremely useful

**Comment by bfops** [comment]
Python support is on our radar! As you can imagine, we're currently juggling various priorities right now.

**Comment by Lynden-Sylvester** [comment]
> Python support is on our radar! As you can imagine, we're currently juggling various priorities right now.

Glad to hear it! And no worries! I'm looking forwards to it once y'all have addresses the higher priority tasks!

Keep up the good work!

**Comment by emm-hhec** [comment]
You guys need a good ai agent support to help you with backlog ...

Upvote on this thread 🆙
